isisd/isis_circuit.c

```diff
@@ -955,6 +955,8 @@ void isis_circuit_print_json(struct isis_circuit *circuit,
 	char buf_prx[INET6_BUFSIZ];
 	char buf[255];
 
+	json_object_int_add(json, "circuit", circuit->circuit_id);
```

Comment:
why add this value a second time, with a new key - isn't it being added just below with "circuit-id"?

Reply:
I want to print the circuit_id only when printing a specific interface, if this modification is allowed.